frontend/src/screens/learner/card.tsx

```diff
@@ -126,150 +118,325 @@ const CompleteFlag: React.FC<StudyCardProps> = ({ study }) => {
     if (!study.completedAt) return null
 
     return (
-        <Box gap
-            align="center"
-            padding="default"
-            css={{
-                backgroundColor: colors.green,
-                position: 'absolute',
-                borderBottomLeftRadius: 20,
-                borderTopLeftRadius: 20,
-                right: 0,
-                top: 16,
-            }}
-        >
-            <Icon icon="checkCircle" color='white' />
-            <span>Complete</span>
-        </Box>
+        <Flex justify='center' align='center' pos='absolute' top='-.5rem' right='-.5rem' style={{ zIndex: 2 }}>
+            <div style={{ width: '2rem', 
```

Comment:
we do have access to this [`paper` component from mantine](https://mantine.dev/core/paper/) which has a shadow property 🤔 i think whenever possible try to use mantine components as we get a consistent look and feel to the app

Reply:
Okay, I'll use the paper component, the Card has utilized the box shadow property, hence I used it here